packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -56,6 +56,7 @@ import 'ovh-angular-responsive-tabs';
 import 'ckeditor';
 import 'script-loader!messenger/build/js/messenger.min.js';
 import 'script-loader!filesize/lib/filesize.js';
+import 'script-loader!angular-websocket/dist/angular-websocket';
```

Comment:
No `// eslint-disable-line` required ?

Reply:
As there are multiple script loader imports, it was disabled for the batch: https://github.com/ovh/manager/blob/3aea938e5b5a4528d80111b29fcbe73271b3ed9b/packages/manager/apps/dedicated/client/app/app.module.js#L1